chainerx_cc/chainerx/routines/manipulation.h

```diff
@@ -22,6 +23,8 @@ Array RollAxis(const Array& a, int8_t axis, int8_t start = 0);
 // Returns a transposed view of the array.
 Array Transpose(const Array& a, const OptionalAxes& axes = nonstd::nullopt);
 
+Array Pad(const Array& a, int8_t pad_width, const std::string& mode, int64_t constant_values);
```

Comment:
Let's use `enum` instead of `std::string`. (just like `chainerx::AveragePoolPadMode` defined in `chainerx/routines/pooling.h`)

(For Python binding, it's ok to use `str`.)

Reply:
Let's use "array" as argument name here, too.

Reply:
I have changed padding mode to use `enum`.